vcr/issuer/issuer.go

```diff
@@ -145,7 +148,7 @@ func (i issuer) Issue(ctx context.Context, credentialOptions vc.VerifiableCreden
 					Info("Wallet or issuer does not support OpenID4VCI, fallback to publish over Nuts network")
 			}
 		}
-		if err := i.networkPublisher.PublishCredential(ctx, *createdVC, public); err != nil {
+		if err := i.networkPublisher.PublishCredential(ctx, *createdVC, options.Public); err != nil {
```

Comment:
I think we should prevent VCs in JWT format to be published on the network?

Reply:
Theoretically, we could support it but there's no practical use case (we won't actively promote it?), so safest to disallow it?